src/pathfinding_service/model/token_network.py

```diff
@@ -101,6 +107,12 @@ def _calculate_fees(self) -> None:
                     balance=Balance(view_in.capacity), amount=total
                 )
                 if fee_in is None:
+                    log.warning(
+                        "Invalid fee calculation (incoming)",
+                        amount=total,
+                        view_out=view_in,
+                        fee_schedule_receiver=view_in.fee_schedule_receiver,
+                    )
```

Comment:
We could stop catching `UndefinedMediationFee` in `backwards_fee_receiver` and `backwards_fee_sender` and then use a single except in `_calculate_fees`:
```python
except UndefinedMediationFee:
    log.warning(...)
    self._is_valid = False
    return
```

Reply:
Yeah, it's time to do that.